pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "PyYAML",
     "requests",
     "toml",
+    "tidecv",         # Needed for TIDE metrics
```

Comment:
I don't think this is needed in dependencies, please remove. Should be fine in just engine.

Reply:
Removed with commit 5f738ed3